Systems/c172p-hydrodynamics.xml

```diff
@@ -433,12 +471,43 @@
             </function>
         </fcs_function>
 
+        <fcs_function name="hydro/rudder-drag-lbs">
+            <function>
+                <description>Drag due to rudders (not reduced by planing)</description>
+                <product>
+                    <value>0.00025</value>
+                    <property>hydro/qbar-psf</property>
+                    <property>hydro/float-beam-ft2</property>
+                    <!--sum>
```

Comment:
What's this commented code doing here? If it's no longer needed then it should be removed.

Reply:
Yes, all the non-descriptive comments need to be removed. I was waiting until I was sure this was the final revision. Thanks for reminding me.